operator/src/main/java/oracle/kubernetes/operator/helpers/BasePodStepContext.java

```diff
@@ -311,4 +314,19 @@ protected boolean isK8sContainer(V1Container c) {
   protected String getMainContainerName() {
     return KubernetesConstants.WLS_CONTAINER_NAME;
   }
+
+  protected boolean isAuxiliaryContainer(V1Container c) {
+    return Optional.ofNullable(c.getName()).map(this::isAuxiliaryContainerName).orElse(false);
+  }
+
+  private boolean isAuxiliaryContainerName(@Nonnull String name) {
+    return name.startsWith(AUXILIARY_IMAGE_INIT_CONTAINER_NAME_PREFIX);
+  }
+
+  protected Optional<V1Container> getAuxiliaryContainer(V1PodSpec v1PodSpec) {
+    return Stream.ofNullable(v1PodSpec.getInitContainers())
+        .flatMap(Collection::stream)
+        .filter(this::isAuxiliaryContainer)
+        .findFirst();
```

Comment:
It looks like this is only checking for the first auxiliary image. There can be multiple auxiliary images such as one aux image for WDT install and one or more images containing WDT models etc. It'll be good to handle changes to all the init container images.

Reply:
Thanks for catching this. Updated code to check for auxiliary images in multiple initcontainers.